### DISCUSSION TITLE
Issues in getting started with mesmo

### DISCUSSION THREAD

**Discussion by ishankholkar** [mesmo-dev/mesmo#23] [Q&A]
hello sir, 

i am working on simulation and optimisation of smart grid with distributed generation as a college research project. I am unable to get started with using mesmo. I have attached below the issue I keep running into. Request you to kindly guide me. I have followed all the steps you have specified for the installation.

(base) C:\Users\Ishan Kholkar>cd C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo

(base) C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo>activate mesmo
The system cannot find the path specified.

C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo>set "GDAL_DRIVER_PATH="

(mesmo) C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo>cd C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo\examples\tutorial

(mesmo) C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo\examples\tutorial>python example_3.py
Writing model file for HiGHS via Gurobi. Please ignore Gurobi license information.
Restricted license - for non-production use only - expires 2023-10-25
2022-10-05 15:54:58,688 | INFO | 'C:\Users\Ishan' is not recognized as an internal or external command,
2022-10-05 15:54:58,688 | INFO | operable program or batch file.
Traceback (most recent call last):
  File "C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo\mesmo\solutions.py", line 1218, in solve_highs
    status = next(line for line in output if line.startswith("Model   status      : "))
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "example_3.py", line 54, in <module>
    main()
  File "example_3.py", line 45, in main
    optimization_problem.solve()
  File "C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo\mesmo\solutions.py", line 1054, in solve
    self.solve_highs()
  File "C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo\mesmo\solutions.py", line 1221, in solve_highs
    raise RuntimeError(f"HiGHS solution status could not be retrieved:\n{command}\n{''.join(output)}")
RuntimeError: HiGHS solution status could not be retrieved:
C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo\highs\bin\highs --model_file C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo\results\temp_2022-10-05_15-54-55_128206\problem.mps --options_file C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo\results\temp_2022-10-05_15-54-55_128206\options.txt --solution_file C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo\results\temp_2022-10-05_15-54-55_128206\solution.txt
'C:\Users\Ishan' is not recognized as an internal or external command,
operable program or batch file.

(mesmo) C:\Users\Ishan Kholkar\Desktop\smart grid simulation\mesmo\examples\tutorial>

**Comment by stroitzsch** [comment]
Hi there, just picking up the work here.

Is this still an issue & should I look into it, or is it out of date?
